test/parallel_api/algorithm/alg.merge/merge.pass.cpp

```diff
@@ -165,5 +175,26 @@ main()
     test_algo_basic_double<std::int32_t>(run_for_rnd_fw<test_non_const<std::int32_t>>());
 #endif
 
+    using T = std::tuple<std::int32_t, std::int32_t>; //a pair (key, value)
```

Comment:
Could you make sure that std::tuple can be used in a kernel when compiling with an oldest supported compiler: Intel® oneAPI DPC++/C++ Compiler 2022.3?

Reply:
I checked some issues on intel-llvm repository related to using std::tuple in kernels and I think, the support was enabled in 2021 major release or even earlier (e.g. https://github.com/intel/llvm/issues/4806 is related to 2021.4)